client/my-sites/stats/summary/index.jsx

```diff
@@ -391,7 +397,9 @@ class StatsSummary extends Component {
 										<DownloadCsv
 											statType={ statType }
 											query={ moduleQuery }
-											path={ path }
+											path={
+												statType === 'statsCountryViews' ? `${ path }-${ geoModeLabel }` : path
```

Comment:
There will be a `countryviews-country` filename for the [legacy Countries module](https://github.com/Automattic/wp-calypso/blob/bb42801b356f24683f2a882b5f3fafb67d0b2cc9/client/my-sites/stats/summary/index.jsx#L152), isn't it?